addons/hr_org_chart/static/src/xml/hr_org_chart.xml

```diff
@@ -0,0 +1,160 @@
+                    class="badge"
+                    tabindex="0"
+                    data-trigger="focus"
+                    t-att-data-emp-name="employee.name"
+                    t-att-data-emp-id="employee.id"
+                    t-att-data-emp-dir-subs="employee.direct_sub_count"
+                    t-att-data-emp-ind-subs="employee.indirect_sub_count"
+                    data-toggle="popover">
+                <t t-esc="employee.indirect_sub_count"/>
+            </span>
+
+            <t t-if="!is_self">
+                <a t-att-href="employee.link" class="o_employee_redirect" t-att-data-employee-id="employee.id">
+                    <h5 class="media-heading"><b><t t-esc="employee.name"/></b></h5>
+                    <strong><t t-esc="employee.job_name"/></strong>
+                </a>
+            </t>
+            <t t-if="is_self">
+                <h5 class="media-heading"><b><t t-esc="employee.name"/></b></h5>
+                <strong><t t-esc="employee.job_name"/></strong>
+            </t>
+        </div>
+    </div>
+</t>
+
+<t t-name="hr_org_chart">
+    <!-- NOTE: Desidered behaviour:
+            The maximun number of people is always 7 (including 'self'). Managers have priority over suburdinates
+            Eg. 1 Manager + 1 self = show just 5 subordinates (if availables)
+            Eg. 0 Manager + 1 self = show 6 subordinates (if available)
+
+        -->
+    <t t-set="emp_count" t-value="0"/>
+
+    <div t-if='widget.emp_data.managers.length &gt; 0' class="o_org_chart_group_up">
+        <t t-if='widget.emp_data.managers_more'>
+            <div class="o_org_chart_entry o_org_chart_more media">
+                <div class="media-left">
+                    <a class="well media-object text-center o_employee_redirect"
+                            t-att-href="widget.emp_data.managers[0].link"
+                            t-att-data-employee-id="widget.emp_data.managers[0].id">
+                        <i t-attf-class="fa fa-angle-double-up"/>
+                    </a>
+                </div>
+            </div>
+        </t>
+
+        <t t-foreach="widget.emp_data.managers" t-as="employee">
+            <t t-set="emp_count" t-value="emp_count + 1"/>
+            <t t-call="hr_org_chart_employee">
+                <t t-set="employee_type" t-value="'manager'"/>
+            </t>
+        </t>
+    </div>
+
+    <t t-if="widget.emp_data.children.length || widget.emp_data.managers.length" t-call="hr_org_chart_employee">
+        <t t-set="employee_type" t-value="'self'"/>
+        <t t-set="employee" t-value="widget.emp_data.self"/>
+    </t>
+
+    <t t-if="!widget.emp_data.children.length &amp;&amp; !widget.emp_data.managers.length">
+        <div class="alert alert-info">
+            <p><b>No hierarchy position.</b></p>
+            <p>This employee has no manager or subordinate.</p>
+            <p>In order get an organigram, assign a manager to the related employees.</p>
```

Comment:
Too late my dear. You will have to fill the form in order to make a fix in master.